apiserver/service/chat.go

```diff
@@ -390,6 +395,7 @@ func (cs *ChatService) PromptStartersHandler() gin.HandlerFunc {
 			c.JSON(http.StatusBadRequest, chat.ErrorResp{Err: err.Error()})
 			return
 		}
+		req.AppNamespace = NamespaceInHeader(c)
```

Comment:
@Abirdcfly  @0xff-dev  Since we all need this namespace in header, can we provide a common way which can be used in all restful apis ?